Objects/TI4_Helpers/TI4_UnitHelper.ttslua

```diff
@@ -920,6 +944,27 @@ local _unitModifiers = {
         end
     },
 
+    ['Ta Zern'] = {
+        description = 'You may reroll any dice. (Automatic tool: Reroll all misses)',
+        leader = LEADER.COMMANDER,
+        owner = OWNER.SELF,
+        type = TYPE.ADJUST,
+        toggleActive = true,
+        apply = function(unitAttrs)
+            for unitType, attr in pairs(unitAttrs) do
+                if attr.spaceCannon then
+                    attr.spaceCannon.rerollMisses = 999 -- Reroll ALL misses
```

Comment:
Shouldn't this be a boolean?  They can't possibly keep rerolling, can they?

Reply:
There is only a single reroll; this controls how many of the original roll's dice can be included in that reroll.

This is to accommodate abilities that say "You may reroll up to N dice". Setting 999 (an arbitrarily high number) would reroll 3 out of 3 misses from a roll, but setting it to 2 for example would only reroll 2 of 3 of those misses.

Reply:
Oh I see, thanks.  Are there other abilities with a reroll dice limit?

Reply:
Not off the top of my head. Easy enough to do this way, though. To accommodate some wonky homebrew.

Reply:
I did go back and just use a boolean. It's cleaner and there's no need for more, yet.